packages/priority-queue/src/test/index.ts

```diff
@@ -29,7 +29,7 @@ describe( 'createQueue', () => {
 			queue.add( {}, callback );
 
 			expect( callback ).not.toHaveBeenCalled();
-			requestIdleCallback.tick();
+			( requestIdleCallback as any ).tick();
```

Comment:
If you replace its import at the top with this, then you will not need to assert as any

```ts
import _requestIdleCallback from '../request-idle-callback';

const requestIdleCallback =
	_requestIdleCallback as typeof _requestIdleCallback & {
		tick: ( deadline?: Partial< IdleDeadline > | number ) => void;
	};
```